README.md

```diff
@@ -36,6 +36,33 @@
 

```

Comment:
Something's not quite right. This is what I see with the default settings:
![screen shot 2017-12-14 at 11 49 17 am](https://user-images.githubusercontent.com/11282622/34012023-073f6d2e-e0c7-11e7-992f-1bc2c4dedb8c.png)
Zoomed in:
![screen shot 2017-12-14 at 11 47 42 am](https://user-images.githubusercontent.com/11282622/34012015-029d961a-e0c7-11e7-91c3-debfc9edfc40.png)
(It's not centered by default and the labels are overlapping)

This is what I see from master:
![screen shot 2017-12-14 at 12 04 39 pm](https://user-images.githubusercontent.com/11282622/34012038-0fa8f1f6-e0c7-11e7-821f-4684cceea1c9.png)

Reply:
Good catch, didn't repro with the test data I was using.

Reply:
Looks better now - thanks